.github/buildomat/jobs/netstack-prepare.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# This script is used to prepare the tools used in the
+# netstack validation environment
+#:
+#: name = "netstack-prepare"
+#: variety = "basic"
+#: target = "helios"
+#: rust_toolchain = "nightly"
+#:
+#: output_rules = [
+#:   "=/out/build-tools.tar.xz",
+#: ]
+#:
+#: access_repos = [
```

Comment:
I think the only access repo that's needed here is testbed. All the other stuff is public.

Reply:
Removed